core/dbt/contracts/sql.py

```diff
@@ -48,7 +42,7 @@ def error(self):
 
 @dataclass
 @schema_version("remote-execution-result", 1)
-class RemoteExecutionResult(ExecutionResult, RemoteResult):
+class RemoteExecutionResult(ExecutionResult):
```

Comment:
Does this mean that the schema for these "remote result" objects has changed and needs to be bumped? Even if the schema has changed, it looks these objects are only ever used by dbt-rpc. If that is indeed the case, can we remove them entirely?